float32[1] joint_pos